File: src/org/funivan/intellij/FastCoddy/Helper/Str.java
package org.funivan.intellij.FastCoddy.Helper;

/**
 * Created by ivan on 03.06.16.
 */
public class Str {

    public static String plural(int num, String singular, String plural) {
        if (num == 1) {
            return Integer.toString(num) + ' ' + singular;
        }
        return Integer.toString(num) + ' ' + plural;
    }

}

File: src/main/kotlin/org/funivan/intellij/FastCoddy/Helper/Str.kt
package org.funivan.intellij.FastCoddy.Helper

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
object Str {
    fun plural(num: Int, singular: String, plural: String): String {
        return if (num == 1) {
            Integer.toString(num) + ' ' + singular
        } else Integer.toString(num) + ' ' + plural
    }
}
